scalafmt-tests/src/test/resources/test/Enum.stat

```diff
@@ -0,0 +1,51 @@
+runner.dialect = dotty
+<<< enum
+enum Color {
+case Red, Green, Blue
+}
+>>>
+enum Color {
+  case Red, Green, Blue
+}
+<<< parameterized enum
+enum Color (val rgb: Int) {
+case Red   extends Color(0xff0000)
+case Green extends Color(0x00ff00)
+case Blue  extends Color(0x0000ff)
+}
+>>>
+enum Color (val rgb: Int) {
+  case Red extends Color(0xff0000)
+  case Green extends Color(0x00ff00)
+  case Blue extends Color(0x0000ff)
+}
+<<< user-defined members of enums
+enum Planet (mass: Double, radius: Double) {
+private final val G = 6.67300e-11
+def surfaceGravity = G * mass / (radius * radius)
+def surfaceWeight(otherMass: Double) = otherMass * surfaceGravity
+
+case Mercury extends Planet(3.303e+23, 2.4397e6)
+case Venus extends Planet(4.869e+24, 6.0518e6)
+case Earth extends Planet(5.976e+24, 6.37814e6)
+case Mars extends Planet(6.421e+23, 3.3972e6)
+case Jupiter extends Planet(1.9e+27, 7.1492e7)
+case Saturn extends Planet(5.688e+26, 6.0268e7)
+case Uranus extends Planet(8.686e+25, 2.5559e7)
+case Neptune extends Planet(1.024e+26, 2.4746e7)
+}
+>>>
+enum Planet (mass: Double, radius: Double) {
```

Comment:
Can you force the line to break and see what indentation you get, and whether it breaks on every comma? Add a few more parameters, to have more commas.

Reply:
Implementation needs changes because it ends up being:
```
enum Planet (mass: Double,
radius: Double,
name: String,
id: Int) {
```

Reply:
In addition, another error occurs (though it doesn't seem to be related to the enum).
```
- test/Enum.stat:24: user-defined members of enums                      | *** FAILED ***
org.scalafmt.Error$FormatterChangedAST: Formatter changed AST
=====================
6.67300e-11 != 6.67300e-11
(scala.meta.Lit$Double$LitDoubleImpl != scala.meta.Term$Block$TermBlockImpl)
```

Reply:
We check if the AST changed after reformatting, which should not happen. This looks suspicious, but might be related to parser. I will take a look.

Reply:
The indentation works as expected for parameter lists now. 
Besides, I threw away the code causing a failing assertion with AST.